### PR TITLE
Relnotes for 0.8.10

### PR DESCRIPTION
**Pull request by sa2ajj**

### CODE REVIEW

File: master/docs/relnotes/0.8.10.rst [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+~~~~~~~~
+
+* Both the P4 source step and P4 change source support ticket-based authentication.
+
+* Clickable 'categories' links added in 'Waterfall' page (web UI).
+
+Fixes
+~~~~~
+
+* Buildbot is now compatible with SQLAlchemy 0.8 and higher, using the newly-released SQLAlchemy-Migrate.
```
Comment (Stibbons) [inline master/docs/relnotes/0.8.10.rst]: Can you point me to the fix(es) that allowed that, I am interested!

**Comment by djmitche** [comment]
```
/home/travis/build/buildbot/buildbot/master/docs/relnotes/0.8.10.rst:29: ERROR: Unknown interpreted text role "bb:bug".
```

File: master/docs/relnotes/0.8.10.rst (hunk 1 above)
Comment (sa2ajj) [inline master/docs/relnotes/0.8.10.rst]: I believe the corresponding change is d313763d

@djmitche do you have any recollection about this particular change?

Comment (djmitche) [inline master/docs/relnotes/0.8.10.rst]: Only that sqlalchemy-migrate 0.8.x works with sqlalchemy-0.8.x.  I don't think we had any other required changes.

Comment (Stibbons) [inline master/docs/relnotes/0.8.10.rst]: Thanks. By the way, I've tested the new version of alembic, when used with the latest sqlalchemy it does marvelous things. I still want to give a try to port buildbot to alembic ;)

**Comment by sa2ajj** [comment]
@djmitche roles were fixed.

**Pull request merged**